core/src/main/java/uk/ac/bbsrc/tgac/miso/core/event/alerter/EmailAlerterService.java

```diff
@@ -60,7 +62,7 @@ public void raiseAlert(Alert a) throws AlertingException {
       throw new AlertingException("No SMTP host specified in the mail.properties configuration file. Cannot send email.");
     } else {
       String from = mailProps.getProperty("mail.from");
-      if (from == null || "".equals(from)) {
+      if (!isStringEmptyOrNull(from)) {
```

Comment:
This one is different! so this should be checking to see if the string is null (as per the removed line).
This change would set the from address to `miso@your.miso.server` as long as it wasn't null.
:confounded: